pkg/kp/kv.go

```diff
@@ -110,7 +91,7 @@ type consulStore struct {
 	podStore podstore.Store
 }
 
-func NewConsulStore(client consulutil.ConsulClient) Store {
+func NewConsulStore(client consulutil.ConsulClient) *consulStore {
```

Comment:
Keeping it private has a nice benefit in that it forces the client, if they ever pass it around, to define the subset of the interface that they need. Accepting a `*consulStore` as a parameter would be way worse, for obvious reasons.